docker-compose.yml

```diff
@@ -7,17 +7,37 @@ services:
       - 9432:9432
     restart: always
 
+  gitcollector:
+    image: srcd/gitcollector:v0.0.1-beta.3
+    # wait for db
+    command: ['/bin/sh', '-c', 'sleep 10s && gitcollector download']
+    environment:
+      GITHUB_ORGANIZATION: ${GITHUB_ORGANIZATION:-}
+      GITHUB_TOKEN: ${GITHUB_TOKEN:-}
+      # use main db
+      GITCOLLECTOR_METRICS_DB_URI: postgresql://superset:superset@postgres:5432/superset?sslmode=disable
```

Comment:
Why do we need to use the main DB? Is it not better to use the metadata db?
I'm assuming at some point we may have a command to cleanup the gitcollector+ghsync data and download again. Removing the metadata db and gitbase_repositories volumes will leave everything in a clean state.

Reply:
I used main db exactly to avoid wiping out stuff related to gitcollector.
I assumed it can update sivas (as it was shown during the demo). But maybe it's not true. I asked for confirmation.

Reply:
> it will update the discovered repositories which are already there, there's is an option to don't allow updates so it would do nothing

confirmed

Reply:
I'm not too sure about this. In my mind it would be better to have gitcollector and ghsync totally in sync.
gitcollector can update sivas, but can it detect if a repo was deleted, or renamed?

Anyway to me this is something worth discussing, but for the initial download it's ok either way.